logger_lib/src/logger_lib.hpp

```diff
@@ -0,0 +1,89 @@
+#ifndef LOGGER_LIB_HPP
```

Comment:
the comment is not for this line, its for the file name
you don't need to add suffix "_lib" to logger_lib.hpp. 
the name logger is sufficient, generally its lib or not such information is not provided by such suffix. the reademe, etc are there to provide such information.

Reply:
Also its better if you split your implementation in .hpp and .cpp
declaration and definition.